### PR TITLE
Completed CSS

### PR DESCRIPTION
**Pull request by swapnil3574**

### CODE REVIEW

File: swapnil/CSS/Theory.txt [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+p{color:red}
+                </style>
+        </head>
+        <body>
+                <div id="section1">
+                        <p> this is first paragraph </p>
+                </div>
+        </body>
+</html>
+
```
Comment (avinash0070) [inline swapnil/CSS/Theory.txt:88]: bad css code indentation 
use tab (4 spaces)

File: swapnil/CSS/Theory.txt [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+}
+                </style>
+        </head>
+        <body>
+                <div id="parent">
+                        <h2>This is heading!</h2>
+                </div>
+        </body>
+</html>
+
```
Comment (avinash0070) [inline swapnil/CSS/Theory.txt:111]: bad css code indentation 
use tab (4 spaces)

File: swapnil/CSS/Theory.txt [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+#s1{
+        padding-left:20px;
+        background:yellow;
+}
+                </style>
+                <body>
+                        <span id="s1"> Amber Academy </span>
+                </body>
+</html>
+
```
Comment (avinash0070) [inline swapnil/CSS/Theory.txt:193]: bad css and code indentation 
use Tab(4 spaces)

Comment (swapnil3574) [reply swapnil/CSS/Theory.txt:193]: done changes

File: swapnil/CSS/Theory.txt (hunk 1 above)
Comment (swapnil3574) [reply swapnil/CSS/Theory.txt:88]: done changes

File: swapnil/CSS/Theory.txt (hunk 2 above)
Comment (swapnil3574) [reply swapnil/CSS/Theory.txt:111]: done changes

File: swapnil/CSS/Theory.txt (hunk 1 above)
Comment (VIPUL2812) [reply swapnil/CSS/Theory.txt:88]: bad indentation from line no. 76 to 80.

File: swapnil/CSS/Theory.txt (hunk 2 above)
Comment (VIPUL2812) [reply swapnil/CSS/Theory.txt:111]: bad indentation from line no. 96 to 103.

File: swapnil/CSS/Theory.txt (hunk 3 above)
Comment (VIPUL2812) [reply swapnil/CSS/Theory.txt:193]: wrong indentation

Comment (VIPUL2812) [reply swapnil/CSS/Theory.txt:193]: @avinash0070  CSS code is given in question itself

Comment (VIPUL2812) [reply swapnil/CSS/Theory.txt:199]: line no. 177 wrong identation

File: swapnil/CSS/Theory.txt (hunk 2 above)
Comment (avinash0070) [reply swapnil/CSS/Theory.txt:117]: use tab (4space) for HTML code also

File: swapnil/CSS/Theory.txt (hunk 3 above)
Comment (avinash0070) [reply swapnil/CSS/Theory.txt:199]: use tab (4space) for HTML close also

File: swapnil/CSS/Theory.txt [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+        		    background:green;
+   			}
+                </style>
+        </head>
+        <body>
+                <span id="s1"> Amber Academy </span>
+                <span id="s2"> Amber Academy </span>
+        </body>
+</html>
+
```
Comment (avinash0070) [inline swapnil/CSS/Theory.txt:167]: use tab (4space) for HTML close also

File: swapnil/CSS/Theory.txt [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+			#s1{
+       			     margin-left:20px;
+      		   	     background:yellow;
+			}
+                </style>
+                <body>
+                        <span id="s1"> Amber Academy </span>
+                </body>
+</html>
+
```
Comment (avinash0070) [inline swapnil/CSS/Theory.txt:184]: line no 176  need proper indentation 
and use tab (4space) for HTML code also

**Pull request merged**